test/elixir_internal_certificate_web/features/search_list/searching_list_test.exs

```diff
@@ -0,0 +1,40 @@
+defmodule ElixirInternalCertificateWeb.SearchList.SearchingListTest do
```

Comment:
is it a search feature, can we name it as just `view_list_keyword`

Reply:
updated to `view_keyword_list` module in [1889475](https://github.com/nvminhtue/elixir_internal_certificate/pull/36/commits/1889475a2a989200399e58c18afa287a31772e39)